<?xml version="1.0"?>

<launch>

  <node ns="object_categorization" name="object_categorization" pkg="cob_object_categorization" type="object_categorization" output="screen">
    <!--param name="ip_address" type="str" value="192.168.21.102"/-->
    <remap from="input_pointcloud_segments" to="/object_segmentation/segmented_object"/>
    <remap from="input_color_image" to="/camera/rgb/image_color"/>
    <remap from="input_pointcloud_camera_info" to="/camera/depth_registered/camera_info"/>
  </node>

</launch>
